modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -5,11 +5,16 @@
 // TODO: Same as other module - Better procedure heading that tells you what this is doing
 
 
-The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage {product-title} into a customer’s Google Cloud Platform (GCP) project. Red Hat requires several prerequisites in order to provide these services.
+The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
 
 [WARNING]
 ====
-To use {product-title} in your GCP project, the GCP organizational policy constraint, `constraints/iam.allowedPolicyMemberDomains`, cannot be in place.
+To use {product-title} in your GCP project, the following GCP organizational policy constraints cannot be in place:
+
+* `constraints/iam.allowedPolicyMemberDomains`
+* `constraints/storage.uniformBucketLevelAccess`
+* `constraints/compute.restrictLoadBalancerCreationForTypes`
+* `constraints/compute.vmExternalIpAccess` (This policy constraint is unsupported only during installation. Re-enable the policy constraint after installation.)
```

Comment:
Just a question: Is it required for you to re-enable the `constraints/compute.vmExternalIpAccess` policy constraint. What do you think of adding
```suggestion
* `constraints/compute.vmExternalIpAccess` (This policy constraint is unsupported only during installation.  You can re-enable the policy constraint after installation.)
```
The sentence is a little awkward as is. You could also say
```suggestion
* `constraints/compute.vmExternalIpAccess` (This policy constraint is unsupported only during installation. Ensure to re-enable the policy constraint after installation.)
```

Reply:
Probably just "you can re-enable".